src/idioms/ctor.md

```diff
@@ -3,9 +3,9 @@
 ## Description
 
 Rust does not have constructors as a language construct. Instead, the
-convention is to use an [associated function][] `new` to create an object:
+convention is to use an [associated function][associated function] `new` to create an object:
 
-```rust
+````rust
```

Comment:
why 4 `?

Reply:
Having `rust` afterwards is the standard way of specifying the language in markdown.
See [this](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)

Reply:
I have never seen 4 `. Can we go back to 3?